deterministic-bloom/tests/integration_test.rs

```diff
@@ -1,4 +1,16 @@
+use deterministic_bloom::{test_utils::Rvg, BloomFilter};
+
 #[test]
-fn test_add() {
-    assert_eq!(deterministic_bloom::add(3, 2), 5);
+fn test_contains() {
+    let mut bloom = BloomFilter::<256, 30>::new();
+    let mut rvg = Rvg::new();
+
+    let new_val: Vec<u8> = vec![rvg.sample(&(..255u8))];
+    bloom.insert(&new_val);
+
+    for _ in 1..25 {
+        bloom.insert(&vec![rvg.sample(&(..255u8))]);
+    }
+
+    assert!(bloom.contains(&new_val));
 }
```

Comment:
Would proptest macro work better here? I'm thinking @zeeshanlakhani wrote rvg for benches.
Just because the proptest macro is going to be more flexible.

```rs
// Not tested.

use test_strategy::proptest;
use proptest::collection::vec;

#[proptest(cases = 1000)]
fn test_contains(
    #[strategy(vec(vec(0..256u8, 0..100), 26))] 
    values: Vec<Vec<u8>>
) {
    let mut bloom = BloomFilter::<256, 30>::new();

    for v in vals {
        bloom.insert(&v);
    }
    
    assert!(bloom.contains(&vals[0]));
}
```

Reply:
yeah, purely for tests, we should abuse the proptest macro itself for something like this.